exporter/sematextexporter/config.go

```diff
@@ -45,6 +49,12 @@ type MetricsConfig struct {
 	// PayloadMaxBytes is the maximum number of line protocol bytes to POST in a single request.
 	PayloadMaxBytes int `mapstructure:"payload_max_bytes"`
 }
+type LogsConfig struct {
+	// App token is the token of Sematext Monitoring App to which you want to send the logs.
+	AppToken string `mapstructure:"app_token"`
+	// LogsEndpoint specifies the endpoint for receiving logs in Sematext
+	LogsEndpoint string `mapstructure:"logs_endpoint"`
```

Comment:
Similar to `MetricsEndpoint` in `MetricsConfig`, this should not be mapped to YAML config, right?
```suggestion
	LogsEndpoint string `mapstructure:"-"`
```